### PR TITLE
rss fronted button

### PR DESCRIPTION
**Pull request by Colin-Lane**
- adds the button with a modal directing the user what to do with the
  feed url.

@andrewconner @stkem

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/layout/main/main.styl [hunk 1]
```diff
@@ -216,3 +216,19 @@ input::-ms-clear
...
+    font-size: 14px
+    line-height: .2
+
+  .kf-feed-text
+    text-align: left
+
+    label
+      font-weight: bold
+    input
+      width: 100%
```
Comment (stkem) [inline kifi-backend/modules/shoebox/angular/src/layout/main/main.styl]: missing newline at end of file

Comment (Colin-Lane) [inline kifi-backend/modules/shoebox/angular/src/layout/main/main.styl]: :( I'll go back and fix those.

File: kifi-backend/modules/shoebox/angular/src/libraries/library.styl [hunk 2]
```diff
@@ -107,4 +121,4 @@ twoColMinWidth = appSidePad1 + leftColWidth + colGapWidth + rightColWidth + appS
   margin: 0 6px
 
   @media (min-width: 480px)
-    margin: 0 10px
+    margin: 0 10px
```
Comment (stkem) [inline kifi-backend/modules/shoebox/angular/src/libraries/library.styl]: newline missing :-)

File: kifi-backend/modules/shoebox/angular/src/libraries/libraryFeedModal.tpl.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+		<div class="kf-feed-text">
+			<label for="rss">RSS Feed:</label>
+			<input onclick="this.select();" name="rss" type="text" value='{{"https://www.kifi.com/" + username + "/" + librarySlug + "/rss"}}'></input>
+		</div>
+		<div class="kf-feed-text">
+			<label for="atom">Atom Feed:</label>
+			<input onclick="this.select();" name="atom" type="text" value='{{"https://www.kifi.com/" + username + "/" + librarySlug + "/atom"}}'></input>
+		</div>
+	</div>
+</div>
```
Comment (stkem) [inline kifi-backend/modules/shoebox/angular/src/libraries/libraryFeedModal.tpl.html]: newline

**Comment by stkem** [comment]
formatting only comment. Code lgtm.

**Comment by Colin-Lane** [comment]
Sweet, I'll add the newlines, then merge the code.

**Pull request merged**